streams/src/main/java/org/apache/kafka/streams/StreamsMetrics.java

```diff
@@ -58,9 +57,9 @@
      * @param recordingLevel     the recording level (e.g., INFO or DEBUG) for this sensor.
      * @param tags               additional tags of the sensor
      * @return The added sensor.
-     * @see #addRateTotalSensor(String, String, String, RecordingLevel, String...)
+     * @see #addRateTotalSensor(String, String, String, Sensor.RecordingLevel, String...)
      * @see #removeSensor(Sensor)
-     * @see #addSensor(String, RecordingLevel, Sensor...)
+     * @see #addSensor(String, Sensor.RecordingLevel, Sensor...)
```

Comment:
This has been fixed by: https://github.com/apache/kafka/pull/8314
Please omit and rebase to get the latest changes. 

Same for the rest of the issues in this file probably